### DISCUSSION TITLE
añadir opcion de desactivar movimiento automatico

### DISCUSSION THREAD

**Discussion by bombardeo38** [mriscoc/Ender3V2S1#545] [Ideas]
lo primero es agradecerte el friware que haces seria posible añadir la funcion de desconexion del desplazamiento automatico no se si me explicado bien cuando quieres mover un eje se mueve automaticamente encuanto mueves la rueda lo que influye en cuando quieres calibrar los pasos puesto que antes ponias la cantidad de milimetros y le dabas a la rueda y se movia el eje

**Comment by mriscoc** [comment]
It is planned for the next version.
